chalicelib/checks/wrangler_checks.py

```diff
@@ -2424,8 +2425,9 @@ def find_lab(record, all_labs, all_grants):
         score = 0
         best = ''
         log = []
+        matcher = stringmatch.Levenshtein()
         for disp in all_lab_names:
-            s = fuzz.token_sort_ratio(record['OH Lab'], disp.split(',')[0])
+            s = matcher.get_sim_score(record['OH Lab'], disp.split(',')[0]) * 100
```

Comment:
Same potential issues here with `* 100`. Maybe or maybe not OK according to same details as above, I guess.